csi/proto/mayastornodeplugin.proto

```diff
@@ -22,8 +22,15 @@ service MayastorNodePlugin {
   // Unfreeze the file system identified by the volume ID,
   // no check is made if the file system had been previously frozen.
   rpc UnfreezeFS (UnfreezeFSRequest) returns (UnfreezeFSReply) {}
+  // Find the volume identified by the volume ID, and return the mount type:
+  // raw block or filesystem
+  rpc FindVolume (FindVolumeRequest) returns (FindVolumeReply) {}
 }
 
+enum MountType {
+  MOUNT_TYPE_FILESYSTEM = 0; // File system mount
```

Comment:
would VolumeType not be more appropriate? IIRC that's what CSI calls it? In any case, we don't need this long name, MountType::FileSystem and MountType::Block would be sufficient, right?

Reply:
Will change it to VolumeType
If you use MOUNT_TYPE_BLAH, then in the rust code it becomes MountType::blah, see csi/src/nodeplugin_grpc.rs find_volume.

Reply:
Done.